js_modules/dagit/packages/core/src/execute/ConfigEditorConfigPicker.tsx

```diff
@@ -249,18 +251,17 @@ const ConfigEditorConfigGeneratorPicker: React.FC<ConfigEditorConfigGeneratorPic
   (props) => {
     const {configGenerators, label, onSelect} = props;
     const {flagPipelineModeTuples} = useFeatureFlags();
-    const select = React.useRef<Select<ConfigGenerator>>(null);
+    const button = React.useRef<Button | null>(null);
     const itemLabel = flagPipelineModeTuples ? 'Ops' : 'Solids';
 
     return (
       <div>
         <ShortcutHandler
           shortcutLabel={'⌥E'}
           shortcutFilter={(e) => e.keyCode === 69 && e.altKey}
-          onShortcut={() => activateSelect(select.current)}
+          onShortcut={() => button.current?.buttonRef?.click()}
```

Comment:
This is a later todo, but it'd be nice if we used useImperativeRef (or whatever it was called) to expose a .click function on the component ref here to avoid the extra buttonRef layer!